packages/image_picker/image_picker/example/android/app/src/test/resources/robolectric.properties

```
@@ -0,0 +1 @@
+sdk=29
```

nit: missing newline at end of file
```suggestion
sdk=29

```